app-admin/u-boot-menu/spec

```diff
@@ -0,0 +1,3 @@
+VER=4.2.3
+SRCS="git::commit=tags/debian/${VER}::https://salsa.debian.org/debian/u-boot-menu.git"
+CHKSUMS="SKIP"
```

Comment:
Missing newline at end-of-file.

Reply:
You should also add a CHKUPDATE field, see [documentation](https://github.com/AOSC-Dev/aosc-findupdate/tree/master/docs). Prefer Anitya when possible.

Reply:
There is no Anitya ID for `u-boot-menu`, so I will use GitLab's API. You can review whether my format is correct.